src/router.ts

```diff
@@ -13,6 +14,8 @@ export default (): Router<AppState, AppServiceContext> => {
   router.get(Routes.ArticleList, '/articles', articleList());
   router.post(Routes.AddArticle, '/articles', addArticle());
   router.get(Routes.EntryPoint, '/', entryPoint());
+  router.get(Routes.Article, '/articles/:id', article());
```

Comment:
We can't use the router for individual articles, as the values are persistent (the 'route' could change, but the existing values won't).